Parsers/ASimAuditEvent/Parsers/vimAuditEventSentinelOne.yaml

```diff
@@ -0,0 +1,714 @@
+    ];
+    let EventFieldsLookupMachineActivity = datatable(
+        activityType_d: real,
+        Operation: string,
+        EventType_machineactivity: string,
+        EventSubType_machineactivity: string,
+        EventResult: string,
+        Object: string,
+        ObjectType: string
+    )
+        [
+        52, "User Approved Agent Uninstall Request", "Other", "Approve", "Success", "Agent", "Service",
+        53, "User Rejected Agent Uninstall Request", "Other", "Reject", "Failure", "Agent", "Service",
+        54, "User Decommissioned Agent", "Disable", "", "Success", "Agent", "Service",
+        55, "User Recommissioned Agent", "Enable", "", "Success", "Agent", "Service",
+        61, "User Disconnected Agent From Network", "Execute", "", "Success", "Agent", "Service",
+        62, "User Reconnected Agent to Network", "Execute", "", "Success", "Agent", "Service",
+        63, "User Shutdown Agent", "Execute", "", "Success", "Agent", "Service",
+        93, "User Reset Agent's Local Config", "Set", "", "Success", "Local config", "Configuration Atom",
+        95, "User Moved Agent to Group", "Other", "Move", "Success", "Agent", "Service",
+        117, "User Disabled Agent", "Execute", "", "Success", "Agent", "Service",
+        118, "User Enabled Agent", "Execute", "", "Success", "Agent", "Service",
+        4100, "User Marked Deep Visibility Event As Threat", "Set", "", "Success", "Deep Visibility Event", "Other",
+        4101, "User Marked Deep Visibility Event As Suspicious", "Set", "", "Success", "Deep Visibility Event", "Other",
+    ];
+    let EventFieldsLookupAccountActivity = datatable(
+        activityType_d: real,
+        Operation: string,
+        EventType_accountactivity: string,
+        EventSubType_accountactivity: string,
+        EventResult: string,
+        Object: string,
+        ObjectType: string
+    )
+        [
+        130, "Opt-in To EA program", "Create", "", "Success", "", "Other",
+        131, "Opt-out From EA Program", "Delete", "", "Success", "", "Other",
+        5040, "Account Created", "Create", "", "Success", "", "Other",
+        5041, "Account Modified", "Set", "", "Success", "", "Other",
+        5042, "Account Deleted", "Delete", "", "Success", "", "Other",
+        5044, "Account Policy Reverted", "Set", "", "Success", "", "Other",
+        7200, "Add cloud account", "Create", "", "Success", "", "Other",
+        7201, "Disable cloud Account", "Disable", "", "Success", "", "Other",
+        7202, "Enable cloud Account", "Enable", "", "Success", "", "Other"
+    ];
+    let EventFieldsLookup_useractivity = datatable(
+        activityType_d: real,
+        Operation: string,
+        EventType_useractivity: string,
+        EventSubType_useractivity: string,
+        EventResult: string,
+        Object: string,
+        ObjectType: string
+    )
+        [
+        67, "User 2FA Modified", "", "", "Success", "Two Factor Authentication", "Policy Rule",
```

Comment:
Here, the User modified 2fA for himself and does not have a different ActorUser & TargetUser. Hence it's more suitable for Audit instead of UserManagement

Reply:
If user modifies its own properties, we can refer it as TargetUser. I would suggest moving this log condition to UserManagement.

The Microsoft Sentinel user management normalization schema is used to describe user management activities, such as creating a user or a group, changing user attribute, or adding a user to a group. Such events are reported, for example, by operating systems, directory services, identity management systems, and any other system reporting on its local user management activity. https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-user-management

Reply:
Audit logs are more related to system not user.

The audit trail logs system configuration activities and policy changes. Such changes are often performed by system administrators, but can also be performed by users when configuring the settings of their own applications. https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-audit

Reply:
Understood. We are moving the below activity events related to user from Audit schema to User Management and then update the PR for Audit schema and raise new PR with this added event for User Management.
**ActivityType:** 42, 67, 145, 146, 147

Reply:
We have removed 42, 67, 145, 146, and 147 activities from the AuditEvent Parser.